src/includes/configuration/before-send/ruby.rails.mdx

```diff
@@ -0,0 +1,12 @@
+```ruby
+Sentry.init do |config|
+  # this example uses Rails' parameter filter to sanitize the event payload
```

Comment:
I chose this as Rails' example because it's the most requested feature that we don't support (client side data scrubbing).